DXRCore/DeusEx/Classes/DXRInfo.uc

```diff
@@ -8,9 +8,12 @@ var transient int passes;
 var transient int fails;
 var config int config_version;
 
-simulated function class<Actor> GetClassFromString(string classstring, class<Actor> c)
+simulated function class<Actor> GetClassFromString(string classstring, class<Actor> c, optional bool silent)
 {
     local class<Actor> a;
+    local int i;
+    local string orig;
```

Comment:
What are these 2 new local variables for?

Reply:
Oh, those are definitely leftovers.  Related to the Zombie respawning for Revision, I previously had the logic for checking other packages in here.  Now all that logic (and the use of the equivalent of these variables) is in DXRZombies